mlflow/cli.py

```diff
@@ -58,6 +59,11 @@ def cli():
                    "Databricks. See "
                    "https://docs.databricks.com/api/latest/jobs.html#jobsclusterspecnewcluster for "
                    "more info. Note that MLflow runs are currently launched against a new cluster.")
+@click.option("--cluster-spec-json", metavar="STRING",
+              help="JSON string describing the cluster to use when launching a run on "
+                   "Databricks. See "
+                   "https://docs.databricks.com/api/latest/jobs.html#jobsclusterspecnewcluster for "
+                   "more info. Note that MLflow runs are currently launched against a new cluster.")
```

Comment:
Is there any way we can reuse the previous parameter for this? For example, if the string ends in ".json" we treat it as a file and otherwise we parse it at JSON. It might be simpler to use than specifying a new parameter with a longer name.